pyproject.toml

```
@@ -28,6 +28,7 @@ dependencies = [
     "matplotlib",
     "sbpy",
     "tables",
+    "difi"
```

I released a pre-release which apparently by default doesn't get installed with `pip install difi` so I think pinning the version here will fix the import issues. 
```suggestion
    "difi == 1.2rc1"
```